soc/arm/fvp_aem/Kconfig.soc

```diff
@@ -13,9 +13,14 @@ config SOC_FVP_V9A
 	bool
 	select SOC_SERIES_FVP_AEM
 
+config SOC_FVP_A320
+	bool
+	select SOC_SERIES_FVP_AEM
+
 config SOC
 	default "v8a" if SOC_FVP_V8A
 	default "v9a" if SOC_FVP_V9A
+	default "a320" if SOC_FVP_A320
```

Comment:
as above, fix in whole PR. If a comment has been raised, look for the issue in all commits and fix them everywhere, new notices of the same thing I will not report